### PR TITLE
Updated code to show Save and add more button while editing datasets

### PR DESCRIPTION
**Pull request by ashmeet-kandhari**
Fixes #

### Proposed fixes:
1. Moved Save and Add button outside if clause.
2. Using single save block but with different actions
3. Removed the add block

### Features:

- [ ] includes tests covering changes
- [ ] includes updated documentation
- [x] includes user-visible changes
- [ ] includes API changes
- [ ] includes bugfix for possible backport

Please [X] all the boxes above that apply

### CODE REVIEW

File: ckan/templates/package/snippets/resource_form.html [hunk 1]
```diff
@@ -72,16 +72,16 @@
...
-      {% endblock %}
     {% endif %}
+    {% block again_button %}
+      <button class="btn btn-default" name="save" value="again" type="submit">{{ _('Save & add another') }}</button>
+    {% endblock %}
+    {% block save_button %}
+      {% if stage %}
+        <button class="btn btn-primary" name="save" value="go-metadata" type="submit">{{ _('Finish') }}</button>
+      {% else %}
+        <button class="btn btn-primary" name="save" value="go-dataset-complete" type="submit">{{ _('Finish') }}</button>
```
Comment (amercader) [inline ckan/templates/package/snippets/resource_form.html]: Can we keep the "Add" label on this case? I think it makes more sense to keep the existing message.

**Comment by amercader** [comment]
Thanks a lot @ashmeet-kandhari this works as expected, only one minor comment and it's good to go

**Comment by ashmeet-kandhari** [comment]
Hi @amercader i have reverted all the changes and just added the if-else logic now.

**Comment by amercader** [comment]
Thanks @ashmeet-kandhari

**Pull request merged**